components/html-block/test/html-block.test.js

```diff
@@ -135,4 +161,16 @@ describe('d2l-html-block', () => {
 		expect(spans[1].innerHTML).to.equal('second');
 	});
 
+	it('should do multiple renderer replacements', async() => {
```

Comment:
🎉 ... I think this is a good additional test.

What are your thoughts on a unit or visual-diff test that includes both the code formatting and the math rendering? Such a test would at least ensure they are properly returning the element, which is currently a requirement of `d2l-html-block` unless we change the API such that plugin renderers are always required to operate on the same element, and not create a new one... not sure if that is possible though. Of course, renderers that are external to core (like @mentions) could still blow things up and would not get caught here, as previously mentioned elsewhere.

Reply:
I certainly think there's value here. Unit test might be harder because I'm not quite sure how easy it is for us to actually await the rendering before asserting anything. The html-block doesn't currently broadcast when it's done its work. Visual diff is easier since we're inevitably tossing waits around anyways.

Reply:
If visual-diff covers it, I wouldn't create a unit test.